guides/common/modules/proc_enabling-pull-based-transport.adoc

```diff
@@ -0,0 +1,97 @@
+[id="enabling-pull-based-tranport_{context}"]
+= Enabling Pull-based Transport
+
+.Procedure
+If your {Project} or {SmartProxy} is preinstalled, use the following procedure to enable the pull.
+
+. Run
+ifdef::satellite[]
+```
+`satellite-installer --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt`
+```
+endif::[]
+ifdef::foreman,orcharhino,katello[]
+```
+`foreman-installer --foreman-proxy-plugin-remote-execution-script-mode=pull-mqtt`.
```

Comment:
Please don't include the dot. Have a look at the rendered version at https://theforeman-foreman-documentation-preview-pr-1584.surge.sh/nightly/Installing_Proxy/index-katello.html#enabling-pull-based-tranport_smart-proxy